google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -987,6 +987,19 @@ public static QueryResultsOption maxWaitTime(long maxWaitTime) {
   QueryResponse query(QueryRequest request);
```

Comment:
We are indeed open to breaking changes, but I wanted to limit the scope of this current PR because it's under the context of our fix-it week. @vam-google will be making more extensive changes for BigQuery in the coming weeks.